scripts/nightly/upload.py

```diff
@@ -18,16 +19,27 @@
     str(f),
     "releases.wagtail.io",
     "nightly/dist/" + f.name,
-    ExtraArgs={"ACL": "public-read"},
 )
 
 print("Updating latest.json")  # noqa: T201
 
 boto3.resource("s3").Object("releases.wagtail.io", "nightly/latest.json").put(
-    ACL="public-read",
     Body=json.dumps(
         {
             "url": "https://releases.wagtail.org/nightly/dist/" + f.name,
         }
     ),
 )
+
+print("Purging cache")  # noqa: T201
+
+boto3.client("cloudfront").create_invalidation(
+    DistributionId="E283SZ5CB4MDM0",
+    InvalidationBatch={
+        "Paths": {
+            "Quantity": len(2),
```

Comment:
```suggestion
            "Quantity": 2,
```

Or store the items in a list and do `len(items)`